EdFi.Buzz.UI/src/Feature/StudentRoster/studentCard.tsx

```diff
@@ -0,0 +1,291 @@
+  .primary-contact-label {
+    font-size: 14px;
+    color: var(--shark);
+  }
+
+  .relationship {
+    font-size: 14px;
+  }
+
+  :hover {
+    outline-color: var(--denim);
+  }
+
+  .d-flex {
+    display: flex !important;
+  }
+
+  & .overflow-hidden {
+    overflow: hidden !important;
+  }
+
+  & div.card-footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-content: center;
+    font: ${(props) => props.theme.fonts.regular};
+    font-weight: 600;
+    color: var(--denim);
+    background-color: var(--white-lilac);
+    border-top: ${(props) => props.theme.border};
+
+    & div {
+      display: flex;
+    }
+
+    & img {
+      justify-self: center;
+      align-self: center;
+      margin: 10px 10px 10px 10px;
+      width: 14px;
+      height: 8px;
+    }
+  }
+
+  .bold {
+    font-size: 14px;
+    font-weight: 600 !important;
+    line-height: 18px;
+  }
+
+  .clickable {
+    cursor: pointer;
+  }
+
+  div.primary-contact-container {
+    display: flex;
+    flex-direction: row;
+    margin: 5px 5px 5px 5px;
+
+    div.primary-contact-name-container {
+      flex: 2;
+      font-size: 14px !important;
+      font-weight: 600 !important;
+      line-height: 18px;
+      max-width: 65%;
+    }
+
+    div.primary-contact-phone-container {
+      white-space: nowrap;
+      flex: 1;
+    }
+  }
+
+  & a {
+    color: var(--denim);
+  }
+
+  img.email-icon,
+  img.phone-icon {
+    margin: 0px 10px 0px 0px;
+  }
+
+  & img.star-icon {
+    margin: 0px 5px 0px 0px;
+  }
+
+  & h3 {
+    color: var(--shark);
+    font: ${(props) => props.theme.fonts.bold};
+    font-size: 18px;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+  }
+
+  & div.image-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 5px 5px 5px 5px;
+
+    & > img {
+      height: 48px;
+      width: 48px;
+    }
+  }
+
+  .outline-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 3em;
+    border-color: var(--denim) !important;
+    border-style: solid;
+    color:  var(--denim);
+    font-weight: 600;
+    border-radius: 4px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    cursor: pointer;
+  }
+
+  .student-profile-pic {
+    @media(max-width:768px){
+      width: 49px;
+      height: 49px;
+    }
+
+    @media(min-width:769px){
+      width: 98px;
+      height: 98px;
+    }
+  }
+`;
+
+export interface StudentCardComponentProps {
+  student: Student;
+}
+
+export const StudentCard: React.FunctionComponent<StudentCardComponentProps> = (props: StudentCardComponentProps) => {
+  const {student} = props;
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  return (
+    <StyledStudentCard className='card'>
+      <div className='student-card-body p-t-0'>
+        <div className='d-flex p-t-12'>
+          <a href={`#/app/studentDetail/${student.studentschoolkey}`}>
```

Comment:
Use Link object. Remove #/app.
```suggestion
          <Link to={`/studentDetail/${student.studentschoolkey}`}>
```

Reply:
Fixed